app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    part = command.split()
+    first_file = part[1]
+    second_file = part[2]
```

Comment:
You can simplify this code
```suggestion
    cmd, first_file, second_file  = command.split()
```